plugins/net_plugin/net_plugin.cpp

```diff
@@ -1185,9 +1185,9 @@ namespace eosio {
                                        ( "peer", c->peer_name())("error",err.message()));
                                  c->connecting = false;
                                  c->close();
-                                 if (started_sessions == 0) {
+                                 /*if (started_sessions == 0) {
                                    connect (c);
-                                 }
+                                 }*/
```

Comment:
This was also removed in Dan's changes. Do you want to remove it with this PR?

Reply:
I added it here to avoid the spinning on startup. I'm trying to take the smallest set of changes that I can put on the testnet right away to hopefully resolve the network stability issues. I do not want to mess with the new net_api code he also added to his PR. Not at this moment at least.